docs/lineage/airflow.md

```diff
@@ -1,18 +1,73 @@
 # Airflow Integration
 
-DataHub supports integration of 
+DataHub supports integration of
+
 - Airflow Pipeline (DAG) metadata
-- DAG and Task run information as well as 
+- DAG and Task run information as well as
 - Lineage information when present
 
 There are a few ways to enable these integrations from Airflow into DataHub.
 
+## Using Datahub's Airflow lineage plugin (new)
+
+:::caution
```

Comment:
::: note

We recommend you use the lineage plugin if you are on Airflow version >= 2.0.2 or on MWAA with an Airflow version >= 2.0.2 
:::